ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -589,8 +576,14 @@ spec:
         app.kubernetes.io/version: ${LOKI_VERSION}
     spec:
       containers:
-      - args:
-        - "-config.file=/etc/promtail/promtail.yaml"
+      - command:
+        - sh
+        - -c
+        - |
+          promtail \
+            -client.url=http://loki.loki.svc.cluster.local:3100/api/prom/push \
```

Comment:
Use `/loki/api/v1/push` instead of the deprecated prom-prefixed APIs here. This will make it easy for observatorium logs to drop support for the deprecated API in near future.